paimon-spark/paimon-spark-common/src/main/java/org/apache/paimon/spark/SparkGenericCatalog.java

```diff
@@ -85,6 +87,8 @@ public class SparkGenericCatalog extends SparkBaseCatalog implements CatalogExte
 
     private CatalogPlugin sessionCatalog = null;
 
+    private final String sparkCaseInsensitive = "spark.case-insensitive";
```

Comment:
If `case-insensitive` is a catalog conf, it is recommended to set like `spark.sql.catalog.$catalogName.case-insensitive` to use it

Reply:
Thanks，would change like @JingsongLi  suggession.